raft-tech/TANF-app
# Front/119/reports

### This pull request changes...
This adds the initial reports page which should match the mock below.
<img width="843" alt="Screen Shot 2020-12-09 at 10 41 05 AM" src="https://user-images.githubusercontent.com/23202538/101672685-0c479080-3a0b-11eb-816b-8fe7d1277ae3.png">

There should be a route for `Reports` with a corresponding navigation tab that is highlighted when on any variation of `/reports/*`.

There should be a page title of 'TANF Reports' and the browser tab should display that name
There should be a `Select` dropdown that allows the user to select either the year `2020` or `2021` for the ATO.
The label for the `Select` dropdown should read "Fiscal Year (October - September)"
There should be a button to `Begin Report` and upon click of this button the user should be routed to `/reports/[the user's selected year]/upload'. There won't be anything on that page yet but that work will be completed in #429 .

This addresses issue #119 


## TO TEST

*List the steps to test the PR*

Log in to the application and you will be directed to `/edit-profile`.
There should be a navigation tab that reads `Reports` - click on that and you should be rerouted to `/reports` where you will see the new page.

You should be able to choose between 2020 and 2021. When you have made a selection, click on `Begin Report` and you should be rerouted to `/reports/[the year you selected]/upload`.

### This pull request is ready to merge when...

  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)
